third_party/tsl/third_party/llvm/workspace.bzl

```diff
@@ -4,8 +4,8 @@ load("//third_party:repo.bzl", "tf_http_archive")
 
 def repo(name):
     """Imports LLVM."""
-    LLVM_COMMIT = "1936bb81aafdbb3b4c9770a24fc77ba07669bd19"
-    LLVM_SHA256 = "7b519ebd1b17dd59b94e5836b431486ec3e2d020c7799c70ce9ee706d25d3c5d"
+    LLVM_COMMIT = "b10899d869954e1426684cbc20a43d7303075d49"
```

Comment:
Same here, this should not be changed.

Reply:
Thanks for pointing it out. Yes, this is irrelevent one. I have cherry-pick it back.